modules/nw-udn-cr.adoc

```diff
@@ -74,8 +74,8 @@ spec:
 +
 * The `subnets` field is mandatory.
 * The type for the `subnets` field is `cidr` and `hostSubnet`:
-** `cidr` is the cluster subnet and accepts a string value.
-** `hostSubnet` specifies the nodes subnet prefix that the cluster subnet is split to.
+** `cidr` specifies the UDN for your cluster. This parameter accepts a string value.
```

Comment:
@dfitzmau CIDR is equivalent to clusterNetwork of cluster. The IP addresses are handed out from this range to pods in user defined network.  
<pre>
 clusterNetwork:
    - cidr: 10.128.0.0/14
      hostPrefix: 23

</pre>
<pre>
  apiVersion: k8s.ovn.org/v1
  kind: UserDefinedNetwork
  metadata:
    name: l3-network
    namespace: a3
  spec:
    layer3:
      mtu: 1400
      role: Primary
      subnets:
      - cidr: 10.151.0.0/16
        hostSubnet: 24
    topology: Layer3

</pre>